package.json

```diff
@@ -77,7 +77,7 @@
     "fastify": "3.29.5",
     "json5": "2.2.3",
     "loader-utils": "2.0.4",
-    "terser": "5.18.2",
+    "terser": "5.19.0",
```

Comment:
This package is in the resolutions, do we need to update it? cc @mattseddon

Reply:
We do keep updating it